conda/recipes/cuproj/meta.yaml

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) 2018-2023, NVIDIA CORPORATION.
+
+{% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev').lstrip('v') %}
+{% set minor_version = version.split('.')[0] + '.' + version.split('.')[1] %}
+{% set cuda_version = '.'.join(environ['RAPIDS_CUDA_VERSION'].split('.')[:2]) %}
+{% set cuda_major = cuda_version.split('.')[0] %}
+{% set py_version = environ['CONDA_PY'] %}
+{% set date_string = environ['RAPIDS_DATE_STRING'] %}
+
+package:
+  name: cuproj
+  version: {{ version }}
+
+source:
+  git_url: ../../..
+
+build:
+  number: {{ GIT_DESCRIBE_NUMBER }}
+  string: cuda{{ cuda_major }}_py{{ py_version }}_{{ date_string }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
+  script_env:
+    - AWS_ACCESS_KEY_ID
+    - AWS_SECRET_ACCESS_KEY
+    - AWS_SESSION_TOKEN
+    - CMAKE_C_COMPILER_LAUNCHER
+    - CMAKE_CUDA_COMPILER_LAUNCHER
+    - CMAKE_CXX_COMPILER_LAUNCHER
+    - CMAKE_GENERATOR
+    - PARALLEL_LEVEL
+    - SCCACHE_BUCKET
+    - SCCACHE_IDLE_TIMEOUT
+    - SCCACHE_REGION
+    - SCCACHE_S3_KEY_PREFIX=cuproj-aarch64 # [aarch64]
+    - SCCACHE_S3_KEY_PREFIX=cuproj-linux64 # [linux64]
+    - SCCACHE_S3_USE_SSL
+    - SCCACHE_S3_NO_CREDENTIALS
+  ignore_run_exports_from:
+    {% if cuda_major == "11" %}
+    - {{ compiler('cuda11') }}
+    {% endif %}
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    {% if cuda_major == "11" %}
+    - {{ compiler('cuda11') }} ={{ cuda_version }}
+    {% else %}
+    - {{ compiler('cuda') }}
+    {% endif %}
+    - cuda-version ={{ cuda_version }}
+    - ninja
+    - sysroot_{{ target_platform }} {{ sysroot_version }}
+  host:
+    - cuda-version ={{ cuda_version }}
+    - cmake {{ cmake_version }}
+    - cython >=0.29,<0.30
+    - python
+    - rmm ={{ minor_version }}
+    - scikit-build >=0.13.1
+    - setuptools
+    - proj
+    - sqlite
+  run:
+    {% if cuda_major == "11" %}
+    - cudatoolkit
+    {% endif %}
+    - {{ pin_compatible('cuda-version', max_pin='x', min_pin='x') }}
+    - python
+    - rmm ={{ minor_version }}
+    - cupy>=12.0.0
+
+test:            # [linux64]
+  imports:       # [linux64]
+    - cuproj     # [linux64]
+  requires:
+    - cupy>=12.0.0
+    - cuspatial ={{ minor_version }}
```

Comment:
Is cuproj ever going to need a more recent version of cuspatial? IOW do we ever need to use `version` than `minor_version`?

Reply:
I guess I don't understand what you mean. This is probably a copy paste.

Reply:
> to use version than minor_version

Is there a word missing there?

Reply:
I copied this from the `rmm` line just below.

Reply:
`{{% version }}` is the result of git describe --tags, I believe. e.g. `23.08.00a-255-gf1d489b6`, while `{{% minor_version}}` is 23.08. We probably want to test cuproj with the most recent cuspatial version.

Reply:
This should be fine as-is. I don't think changes are needed.